Red & Blue Dragons Update 03: Uniforms

CSLeader opened the pull request:
- Added South Korean fatigues
- Added North Korean fatigues
- Added PLA Type 87 fatigues
- Added NK Woodland fatigues and cap
- Added ROKMC WAVEPAT uniforms and helmet
- Added ROK Granite B uniforms, vests, and helmet
- Added ROC DCCU mockup uniforms
- Updated JSDF uniforms, vests, and helmets.
- Updated PLA uniforms, vests, and helmets
- Added KPA Digital Woodland uniforms, vest, and helmet

CSLeader committed:
Asian factions Update 3: Commit 01

-added South Korean fatigues
-added North Korean fatigues
-added PLA Type 87 fatigues
-added NK Woodland fatigues and cap

CSLeader committed:
Asian factions Update 3: Commit 01

-added South Korean fatigues
-added North Korean fatigues
-added PLA Type 87 fatigues
-added NK Woodland fatigues and cap

CSLeader committed:
Commit 02

- Added ROKMC WAVEPAT uniforms and helmet
- Added ROK Granite B uniforms and helmet
- Added ROC DCCU mockup uniforms
- Updated JSDF uniforms, vests, and helmets.
- Updated PLA uniforms, vests, and helmets
- Added KPA Digital Woodland uniforms, vest, and helmet

CSLeader committed:
Commit 02

- Added ROKMC WAVEPAT uniforms and helmet
- Added ROK Granite B uniforms and helmet
- Added ROC DCCU mockup uniforms
- Updated JSDF uniforms, vests, and helmets.
- Updated PLA uniforms, vests, and helmets
- Added KPA Digital Woodland uniforms, vest, and helmet

CSLeader committed:
ROK Vest

- Added Granite B Modular Vests

diwako committed:
add warnings are errors to build script

diwako commented on sim_extra_gear/config.cpp:
```diff
@@ -2053,6 +2053,42 @@ class CfgVehicles
+			"sim_extra_gear\tex\type65_kpa_modern_co.paa"
+		};
+	};
+	class unif_chi_type65_kpa_dh_camo: unif_chi_type65
```
This one aint working

diwako commented on FP_UniformsAsia/config.cpp:
```diff
@@ -622,21 +622,86 @@ class CfgVehicles
+	};
+	class FP_Rifleman_WAVEPAT: I_soldier_F
+	{
+		_generalMacro="B_Soldier_F";
```
inherits from independent side, but general macro is form bluefor side?

CSLeader committed:
Config fixes

- Removed "unif_chi_type65_kpa_dh_camo" class
- Fixed uniforms having incorrect _generalMacros names.

CSLeader committed:
Merge branch 'kpa-uniforms-update-03' of https://github.com/CSLeader/fp_uniforms into kpa-uniforms-update-03

CSLeader committed:
Readded deleted assets
